apis/r/R/utils-matrixZeroBasedView.R

```diff
@@ -1,16 +1,25 @@
-#' matrixZeroBasedView is a wrapper shim for a matrix or Matrix::sparseMatrix providing
+#' Zero-based Wrapper for Sparse Matrices
+#'
+#' \code{matrixZeroBasedView} is a wrapper shim for a matrix or
+#' \code{\link[Matrix:sparseMatrix]{Matrix::sparseMatrix}} that allows
+#' elemental matrix access using zero-based indices
```

Comment:
As noted on another PR today -- I do prefer to end sentences with periods. In order to avoid redundancy, I won't mark up this PR to that effect.

Reply:
Added in ded4168